config/environments/test.rb

```diff
@@ -7,6 +7,7 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  ENV["SOLR_CORE"] = "blacklight-test"
```

Comment:
what do you think about?
`ENV['SOLR_CORE'] = ENV['SOLR_CORE_TEST'] || 'blacklight-test'`

so it can be customized and is 12factor